packages/hardhat/hardhat.config.ts

```diff
@@ -46,6 +46,16 @@ const config: HardhatUserConfig = {
         enabled: process.env.MAINNET_FORKING_ENABLED === "true",
       },
     },
+    calibrationnet: {
+      chainId: 314159,
+      url: "https://api.calibration.node.glif.io/rpc/v1",
+      accounts: [deployerPrivateKey],
+    },
+    filecoinmainnet: {
+        chainId: 314,
+        url: "https://api.node.glif.io",
+        accounts: [deployerPrivateKey],
+    },
     mainnet: {
```

Comment:
Should probably change this to ethereumMainnet

because then you can just keep `calibration` and `mainnet` for filecoin

Reply:
Done ✅